site/_shortcodes/Aside.js

```diff
@@ -37,6 +36,12 @@ function Aside(content, type = 'note') {
   // @ts-ignore: `this` has type of `any`
   const locale = this.ctx.locale || defaultLocale;
   let text;
+
+  // implement backwards compatibility for old "gotchas" type, rewrite to "important" type
+  if (type === 'gotchas') {
+    type = 'important';
+  }
```

Comment:
Good approach! Nevertheless a follow-up PR actually updating the using articles and removing this compat condition would be great to keep things tidy 🙂

Reply:
hey @matthiasrohmer cheers, happy to do that in this one if you like!